packages/web-api/src/methods.ts

```diff
@@ -1407,4 +1413,27 @@ export interface ViewsUpdateArguments extends WebAPICallOptions, TokenOverridabl
   hash?: string;
 }
 
+  /*
+   * `workflows.*`
+   */
+export interface WorkflowsStepCompletedArguments extends WebAPICallOptions, TokenOverridable {
+  workflow_step_execute_id: string;
+  outputs?: object;
+}
+ 
+export interface WorkflowsStepFailedArguments extends WebAPICallOptions, TokenOverridable {
+  workflow_step_execute_id: string;
+  error: object;
```

Comment:
@misscoded We might be able to expand on the error object here now that we know it is required.

```suggestion
  error: {
      message: string;
  };
```